Modify kerberos deployment settings

lipppppp opened the pull request:
The kerberos deployment document exist some problems.
1.New principal should have correct permission to execute actions on HDFS when set the smartsever and smartagent principal.
2.The smart-site.xml should be updated on each agent node.

lipppppp committed:
modify kerberos deployment settings

philo-he commented on docs/enable-kerberos.md:
````diff
@@ -75,15 +76,16 @@ kadmind
+kadmin.local:xst -k /xxx/xxx/smartserver-{hostname}.keytab {username}/{hostname}
 ```
-Note: replace the hostname with the hostname of the smart server
+**Note:** replace the username with the HDFS user who has the correct permission to execute actions and replace hostname with hostname of the smart server
````
I am not sure what "HDFS user" means. Is it the user configured in hadoop-env.sh? I forget why we used smartserver or smartagent in previous tests.

philo-he replied:
Can the username for smart agent and smart server be the same one here?

lipppppp replied:
The user performing the operation changed to smartagent or smartserver after completing kerberos configuration. But these two users do not have permission to operate on some specific files. “HDFS user” means it is best to have the highest operating authority for HDFS like the user configured in dfs.permissions.superusergroup.  
![image](https://user-images.githubusercontent.com/32612402/76483184-4b7f8b80-6451-11ea-9029-2e50e99dbaf9.png)

lipppppp replied:
Of course, we can set this "HDFS user" as other users name. But we should give the user proper authority by Ranger or  other authorization management application.

philo-he replied:
OK, please also use {username} to replace "smartagent"/"smartserver" for setting the value of _smart.server.kerberos.principal/smart.agent.kerberos.principal_ in the below configuration template of smart-site.xml.

philo-he replied:
i.e., line 119, 127

lipppppp replied:
Thanks. I have replaced "smartagent"/"smartserver" in the smart-site.xml.

philo-he commented on docs/enable-kerberos.md:
```diff
@@ -75,15 +76,16 @@ kadmind
+**Note:** replace the username with the HDFS user who has the correct permission to execute actions and replace hostname with hostname of the smart server
 
-### 2.2 Add smartagent Kerberos principals to database and export it to keytabs. Please create principals for each agent.
+### 2.2 Add smartagent Kerberos principals to database and export it to keytabs. Please create principals for each agent.Then use 'scp' to copy each keytab file to each agent.
```
A minor format issue, please keep a space after "." or any other punctuation.

lipppppp replied:
OK, I will add a space after ".".

lipppppp committed:
Update enable-kerberos.md

use {username} to replace "smartagent"/"smartserver" in the configuration template of smart-site.xml and fix the format issue

philo-he commented:
The patch looks good to me. Will commit it and ignore the irrelevant build error. Thanks @lipppppp again!
